deploy/kubernetes/base/controller-serviceaccount.yaml

```diff
@@ -38,22 +38,6 @@ rules:
     verbs: ["get", "watch", "list", "delete", "update", "create"]
 ---
 # Source: aws-fsx-csi-driver/templates/controller-serviceaccount.yaml
-kind: ClusterRoleBinding
```

Comment:
`make generate-kustomize` made these changes. Possible I neglected to run it before. opened https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1137 to update the release docs so it's not neglected